src/cli/scaffold/rust/Cargo.toml.template

```diff
@@ -0,0 +1,14 @@
+[package]
+  name = "code"
+  version = "0.1.0"
+  authors = <<AUTHORS>>
+  edition = <<EDITION>>
+[dependencies]
+  serde = "1.0"
+  serde_json = "1.0"
+  hdk = { git = "https://github.com/holochain/hdk-rust" }
```

Comment:
So this line needs to change. Look at app-spec-rust for the right hdk reference now